microcosm_pubsub/message.py

```diff
@@ -16,12 +16,14 @@ def __init__(self,
                  media_type,
                  message_id,
                  receipt_handle,
+                 topic_arn=None,
```

Comment:
I think you also need to add this to the `parse_raw_message()` in microcosm_pubsub/envelope.py